docs: update nginx config adding breaking changes (#115)
coreruleset/modsecurity-docker: README.md

3. Visit http://localhost:8080 and your page.

### Nginx based images breaking change

| ⚠️ WARNING          |
|:---------------------------|
| Nginx based images are now based on upstream nginx. This changed the way the config file for nginx is generated.  |

If using the [Nginx environment variables](https://github.com/coreruleset/modsecurity-docker#nginx-env-variables) is not enough for your use case, you can mount your own `nginx.conf` file as the new template for generating the base config.

An example can be seen in the [docker-compose](https://github.com/coreruleset/modsecurity-docker/blob/master/docker-compose.yml) file.

> > 💬 What happens if I want to make changes in a different file, like `/etc/nging/conf.d/default.conf`?

Reviewer: ```suggestion
> 💬 What happens if I want to make changes in a different file, like `/etc/nginx/conf.d/default.conf`?
```

> > You mount your local file, e.g. `nginx/default.conf` as the new template: `/etc/nginx/templates/conf.d/default.conf.template`. You can do the similar to other files. Files in the templates directory will be copied preserving the subdirectories structure.

Reviewer: ```suggestion
> You mount your local file, e.g. `nginx/default.conf` as the new template: `/etc/nginx/templates/conf.d/default.conf.template`. You can do this similarly with other files. Files in the templates directory will be copied and subdirectories will be preserved.
```

| SSL_CIPHER_SUITE | A string indicating the cipher suite to use. Uses OpenSSL [list of cipher suites](https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_ciphersuites.html) (Default: `"ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384"` |
| SSL_ENGINE  | A string indicating the SSL Engine Operation Switch (Default: `off`) | 
| SSL_HONOR_CIPHER_ORDER | A string indicating if the server should [honor the cipher list provided by the client](https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslhonorcipherorder) (Allowed values: `on`, `off`. Default: `off`) |
    # If you need to modify nginx config, mount your volume as the template file
    # volumes:

>     #  - ./nginx/nginx.conf:/etc/nginx/templates/nginx.conf.template:rw

Reviewer: This is in the wrong place, isn't it?

Reviewer 2: Definitely. Bad paste :)

Author: Yes, bad paste.
